fontforgeexe/bitmapview.c

```diff
@@ -734,34 +749,31 @@ static void BVExpose(BitmapView *bv, GWindow pixmap, GEvent *event ) {
     if ( bv->showgrid ) {
 	if ( bv->scale>2 ) {
 	    for ( i=bv->xoff+bv->scale; i<bv->width; i += bv->scale )
-		GDrawDrawLine(pixmap,i,0, i,bv->height,0xa0a0a0);
+		GDrawDrawLine(pixmap,i,0, i,bv->height,grid_color);
 	    for ( i=bv->xoff-bv->scale; i>0; i -= bv->scale )
-		GDrawDrawLine(pixmap,i,0, i,bv->height,0xa0a0a0);
+		GDrawDrawLine(pixmap,i,0, i,bv->height,grid_color);
 	    for ( i=-bv->yoff+bv->height-bv->scale; i>0; i -= bv->scale )
-		GDrawDrawLine(pixmap,0,i,bv->width,i,0xa0a0a0);
+		GDrawDrawLine(pixmap,0,i,bv->width,i,grid_color);
 	    for ( i=-bv->yoff+bv->height+bv->scale; i<bv->height; i += bv->scale )
-		GDrawDrawLine(pixmap,0,i,bv->width,i,0xa0a0a0);
+		GDrawDrawLine(pixmap,0,i,bv->width,i,grid_color);
 	}
-	GDrawDrawLine(pixmap,0,-bv->yoff+bv->height-0*bv->scale,bv->width,-bv->yoff+bv->height-0*bv->scale,0x404040);
+	GDrawDrawLine(pixmap,0,-bv->yoff+bv->height-0*bv->scale,bv->width,-bv->yoff+bv->height-0*bv->scale,guide_color);
 	GDrawDrawLine(pixmap,0,-bv->yoff+bv->height-bv->bdf->ascent*bv->scale,
-		bv->width,-bv->yoff+bv->height-bv->bdf->ascent*bv->scale,0x404040);
+		bv->width,-bv->yoff+bv->height-bv->bdf->ascent*bv->scale,guide_color);
 	GDrawDrawLine(pixmap,0,-bv->yoff+bv->height+bv->bdf->descent*bv->scale,
-		bv->width,-bv->yoff+bv->height+bv->bdf->descent*bv->scale,0x404040);
-	GDrawDrawLine(pixmap,bv->xoff+0*bv->scale,0, bv->xoff+0*bv->scale,bv->height,0x404040);
-	GDrawDrawLine(pixmap,bv->xoff+bv->bc->width*bv->scale,0, bv->xoff+bv->bc->width*bv->scale,bv->height,widthcol);
+		bv->width,-bv->yoff+bv->height+bv->bdf->descent*bv->scale,guide_color);
+	GDrawDrawLine(pixmap,bv->xoff+0*bv->scale,0, bv->xoff+0*bv->scale,bv->height,guide_color);
+	GDrawDrawLine(pixmap,bv->xoff+bv->bc->width*bv->scale,0, bv->xoff+bv->bc->width*bv->scale,bv->height,width_guide_color);
 	if ( bv->bdf->sf->hasvmetrics )
 	    GDrawDrawLine(pixmap,0,-bv->yoff+bv->height-(bv->bdf->ascent-bc->vwidth)*bv->scale,
-		    bv->width,-bv->yoff+bv->height-(bv->bdf->ascent-bc->vwidth)*bv->scale,widthcol);
+		    bv->width,-bv->yoff+bv->height-(bv->bdf->ascent-bc->vwidth)*bv->scale,width_guide_color);
     }
     if ( bv->showfore ) {
 	/* Reference names are drawn after grid (otherwise some characters may get unreadable */
 	for ( bref=bc->refs; bref!=NULL; bref=bref->next )
 	    BVDrawRefName( bv,pixmap,bref,0 );
     }
     if ( bv->showoutline ) {
-	Color col = (view_bgcol<0x808080)
```

Comment:
"For the record" I read this as switching from a run-time heuristic to letting the user pick a color, good or bad, and that seems fine to me.